package.json

```diff
@@ -2,10 +2,14 @@
   "name": "express-app",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "server.ts",
   "scripts": {
-    "start": "nodemon index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "nodemon src/server.ts",
+    "test": "npm test",
+    "build": "rimraf ./build && tsc",
+    "start:prod": "npm run build && node build/server.ts",
+    "lint": "eslint . --ext .ts",
+    "lint-and-fix": "eslint . --ext .ts --fix"
```

Comment:
lint:fix name would be more consistent, but I'm not a fan of this commit at all - basically you allow a script to edit your code :/